pom.xml

```diff
@@ -1125,6 +1138,7 @@
       <modules>
         <module>client-apis/jaxrs-client-api</module>
         <module>client-apis/mp_metrics_cdi_extension</module>
+        <module>client-apis/mp_fault_tolerance_cdi_extension</module>
```

Comment:
The CDI extension doesn't need to be referenced twice, it's separate profiles that are combined